components/generic/DownloadButton.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <b-button
+    :href="downloadUrl"
+    :disabled="downloadDisabled"
+    variant="light text-decoration-none"
+    data-qa="download button"
+    size="lg"
+    class="download-button mr-lg-3 d-inline-flex align-items-center"
+    :target="!useProxy ? '_blank' : '_self'"
+  >
+    <span class="ic-download d-inline-flex pr-1"></span>
+    {{ $t('actions.download') }}
+  </b-button>
+</template>
+
+<script>
+  export default {
+    props: {
+      downloadUrl: {
+        type: String,
+        default: ''
+      },
+      downloadDisabled: {
```

Comment:
Rename this to `disabled`. It's implied that it's the download that's disabled given this is the download button component.

Reply:
ok